packages/dd-trace/src/plugin_manager.js

```diff
@@ -71,6 +72,9 @@ module.exports = class PluginManager {
     }
     if (!this._tracerConfig) return // TODO: don't wait for tracer to be initialized
 
+    // TODO: move this somewhere else?
+    if (name === 'kafkajs') this._latencyProcessor = new LatencyStatsProcessor(this._tracerConfig)
```

Comment:
Only use single line conditions for early returns since otherwise it's difficult to see what it does.

Reply:
Also, when we chatted about this I didn't realize there would be a condition for one specific plugin. In that case it might be better to just have this in the plugin itself.

Reply:
@tlhunter Any thoughts since you also seemed to think the plugin manager might not be the best place?